images/manageiq-base-worker/container-assets/manageiq_liveness_check

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
 
-[[ -s /etc/default/evm ]] && source /etc/default/evm
-ruby ${APP_ROOT}/lib/workers/bin/heartbeat_check.rb --heartbeat-file=${APP_ROOT}/tmp/worker.hb
+# We're alive if the hb_file exists and has a unix time "newer" / greater
+# than the current time.
+hb_file="${APP_ROOT}/tmp/worker.hb"
```

Comment:
suggest `heartbeat_file` instead of `hb_file`

Reply:
sure, it was named to match the name of the file.